### PR TITLE
[makefile] Allow building devel files

### PR DESCRIPTION
**Pull request by manuelnaranjo**
Allow building devel non minified versions of the library so debugging is easier for non core developers, IMO this files should be part of the upstream package. Sometimes it helps to see what's going on underneath the courtains.

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Thanks for this PR. However, I think we should embed non minified sources with every build. This way, developers would be able to debug more easily their application without needing to clone this project.

As a consequence, can you update the `build` task to make a `webpack -p` AND a `webpack` of the sources?

**Comment by manuelnaranjo** [comment]
@jpetitcolas it's done now `make build` depens on devel getting both versions at the same time

File: webpack.config.js [hunk 1]
```diff
@@ -1,13 +1,22 @@
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 function getEntrySources(sources) {
-    if (process.env.NODE_ENV !== 'production') { // for live reload
+    if (process.env.NODE_ENV !== 'production' &&
+        process.env.NODE_ENV !== 'development') { // for live reload
```
Comment (jpetitcolas) [inline webpack.config.js]: We need the live reload in development.

**Comment by manuelnaranjo** [comment]
Ok then, I will make a different env called non-minified what do you think? so it doesn't collide with production or development

**Comment by jpetitcolas** [comment]
IMO, a better solution would be:

* Change the `build` command to do both `webpack` and `webpack -p` commands
* Update the `webpack.config.js` file to update filenames: `ng-admin.min.js` if using the production flag, `ng-admin.js` otherwise

To detect the `-p` flag, you can use `process.argv`, as pinpointed by @bebraw on the [Webpack Gitter](https://gitter.im/webpack/webpack). :)

This way, you wouldn't need any extra environment.

**Comment by jpetitcolas** [comment]
Looks good to me. Can you rebase, to ensure tests are passing?

By the way, do not hesitate to ping us when your PR has been updated. I wasn't aware you made some changes. :)

**Comment by jpetitcolas** [comment]
Fixes #614

**Comment by fzaninotto** [comment]
I've tested this PR, and the non-minified files are not helping, because they are not readable. Since we use babel for transpiling ES6 scripts, the non-minified code is already hard to read.

The good solution to your problem, I think, are sourcemaps.

**Comment by manuelnaranjo** [comment]
sourcemaps are way better, I've been using a wiredep solution now, I will create a reference proyect soon and share.

Yet sourcemap are not been provided, not even generated by the current setup.

**Comment by fzaninotto** [comment]
I'll work on a sourcemap setup.

**Comment by fzaninotto** [comment]
Superseded by #619

**Pull request closed without merging**